demos/3d_mode.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>4D Geometry Viewer</title>
+	<script src="../mathbox-src/build/mathbox-bundle.js"></script>
+  	<link rel="stylesheet" href="../../build/mathbox.css">
+  	<script src="../datgui/dat.gui.js"></script>
+  	<meta name="viewport" content="initial-scale=1, maximum-scale=1">
+  	<!-- http://silentmatt.com/javascript-expression-evaluator/ -->
+	<script src="../parser.js"></script>
+</head>
+<body>
+<script type="text/javascript">
+	var colors = {
+		'viewing_axis':'#e59a1c',
+		'data':'#3090FF'
+	}
+	var params = {} //gui params
+	var view;
+	var equation = {
+		eq: null,
+		parse: function(str){
+			//Takes a string and stores the new equation
+			// In case it's a malformed equation, set eq to null so we know something went wrong
+			this.eq = Parser.parse( str ).toJSFunction( ['x'] );
```

Comment:
3d_mode.html is a close copy of 2d_mode.html. The only reason it is formatted that way is that is how 2d_mode.html does it.